Octokit.Tests.Integration/Clients/RepositoryContentsClientTests.cs

```diff
@@ -222,6 +222,18 @@ public async Task GetsDirectoryContent()
                 Assert.Equal(ContentType.Dir, contents.First().Type);
             }
 
+            [IntegrationTest]
+            public async Task GetsContentNonMasterWithRootPath()
+            {
+                var github = Helper.GetAuthenticatedClient();
+
+                var contents = await github
+                    .Repository
+                    .Content
+                    .GetAllContentsByRef("octocat", "Spoon-Knife", "/", reference: "test-branch");
```

Comment:
The URL that gets generated in here looks like this:

https://github.com/octokit/octokit.net/blob/d16ac98d00780f86883133eabad100fc13201e00/Octokit/Helpers/ApiUrls.cs#L2353-L2356

I wonder if the `"/"` here is causing the confusing URL. @SimonCropp what happens if you make this an empty string?

Reply:
null and empty string throw an arg exception. https://github.com/octokit/octokit.net/blob/master/Octokit/Clients/RepositoryContentsClient.cs#L103

happy to attempt a PR where either null or empty string mean "root dir"